doc/api/crypto.md

```diff
@@ -1114,8 +1114,10 @@ module):
 added: v0.7.5
 -->
 
-The `DiffieHellmanGroup` class takes a well-known modp group as its argument but
-otherwise works the same as `DiffieHellman`.
+The `DiffieHellmanGroup` class takes a well-known modp group as its argument.
+It Works the same as `DiffieHellman`, except that it does not allow changing
+its keys after creation - i.e. it does not implement `setPublicKey` or
```

Comment:
```suggestion
its keys after creation. In other words, it does not implement `setPublicKey()` or
```

Reply:
If you do this suggestion, you'll need to re-wrap because this line is now over 80 chars.